### DISCUSSION TITLE
Set focus on first field when subview is a grid.

### DISCUSSION THREAD

**Discussion by CarolineDenis** [specify/specify7#5829] [Feature Requests]
See https://github.com/specify/specify7/pull/3317

**Comment by maxpatiiuk** [comment]
FYI, instead of linking to an entire pull request, you could link to a specific comment with relevant info so that someone looking at this in the future can find out what's happening easier.

In this case that's https://github.com/specify/specify7/pull/3317#issuecomment-1593975969

**Comment by grantfitzsimmons** [comment]
When adding a record in a subform that is displayed in grid view, the first field on that subform should automatically be selected.

**Comment by grantfitzsimmons** [comment]
Should be fixed by #3668 once we finish it
